src/utils/__tests__/quotesService.test.js

```diff
@@ -0,0 +1,77 @@
+import { FRONT_END_ROLE, BACK_END_ROLE, DEV_OPS_ROLE } from '../../constants/roles';
+import { BRAG, CONFESS } from '../../constants/roleActions';
+
+import { getQuote, getRandomQuoteID } from '../quotesService';
+
+jest.mock('../../quotes/backEndQuotes.js', () => ({
```

Comment:
Maybe it would be good idea to actually keep these functions pure and pass data as last argument?

Reply:
I don't get it. Can you elaborate?

Reply:
`function getQuote(role, action, quoteID, quotes)` - that way no need to mock deps